spec.html

```diff
@@ -771,6 +771,27 @@ <h1>Implicit Completion Values</h1>
           1. Return NormalCompletion(*undefined*).
         </emu-alg>
         <p>Any reference to a Completion Record value that is in a context that does not explicitly require a complete Completion Record value is equivalent to an explicit reference to the [[Value]] field of the Completion Record value unless the Completion Record is an abrupt completion.</p>
+
+        <p>Invocations of abstract operations may be prefixed by `!` to indicate that the result will never be an abrupt completion and that the resulting Completion Record's [[Value]] field should be used in place of the return value of the operation.</p>
```

Comment:
I don't think the "!" stuff fits very well in the "Implicit Completion Values" clause. It makes more sense where it is now, in the same clause as "?". Instead of "ReturnIfAbrupt Shorthands", that clause could be called "Shorthands relating to completion records" or just "? and ! Shorthands".

Or split them into 2 clauses: "The ? Shorthand" and "The ! Shorthand".